debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -395,6 +397,10 @@ public boolean getSanitizeFieldNames() {
         }
     }
 
+    public boolean includeSchemaChangeRecords() {
```

Comment:
Can you add a comment? I know it wasn't there before, but it's a good practice for those framework-level classes, and potential future implementors might wonder about its purpose.

Reply:
Maybe we also should rename it, e.g. `isSchemaChangesHistoryEnabled()` or similar?

Reply:
Done